java/jdbc/ConnectionSamples/ADBQuickStart.java

```diff
@@ -0,0 +1,149 @@
+  // TNS_ADMIN - Should be the path where the client credentials zip file is downloaded. 
+  // wallet_dbname - This is the TNS Alias that you can get from tnsnames.ora.
+  final static String DB_URL="jdbc:oracle:thin:@wallet_dbname?TNS_ADMIN=/Users/test/wallet_dbname/";
+  // Update the Database Username to point to your Autonomous Database. For a quick testing, you can 
+  // use "admin" user. 
+  final static String DB_USER = "admin";
+  // For security reasons, you will need to provide database password through console.  
+  static String DB_PASSWORD ;
+  final static String CONN_FACTORY_CLASS_NAME="oracle.jdbc.pool.OracleDataSource";
+
+  /*
+   * The sample demonstrates UCP as client side connection pool.
+   */
+  public static void main(String args[]) throws Exception {
+    // For security purposes, you must enter the password through the console 
+    try {
+      Scanner scanner = new Scanner(System.in);
+      System.out.print("Enter the password for Autonomous Database: ");
+      DB_PASSWORD = scanner.nextLine();
+    }
+    catch (Exception e) {
+       System.out.println("ADBQuickStart - Exception occurred : " + e.getMessage());
+    }
+    
+    // Get the PoolDataSource for UCP
+    PoolDataSource pds = PoolDataSourceFactory.getPoolDataSource();
+
+    // Set the connection factory first before all other properties
+    pds.setConnectionFactoryClassName(CONN_FACTORY_CLASS_NAME);
+    pds.setURL(DB_URL);
+    pds.setUser(DB_USER);
+    pds.setPassword(DB_PASSWORD);
+    pds.setConnectionPoolName("JDBC_UCP_POOL");
+
+    // Default is 0. Set the initial number of connections to be created
+    // when UCP is started.
+    pds.setInitialPoolSize(5);
+
+    // Default is 0. Set the minimum number of connections
+    // that is maintained by UCP at runtime.
+    pds.setMinPoolSize(5);
+
+    // Default is Integer.MAX_VALUE (2147483647). Set the maximum number of
+    // connections allowed on the connection pool.
+    pds.setMaxPoolSize(20);
+
+    // Get the database connection from UCP.
+    try (Connection conn = pds.getConnection()) {
+      System.out.println("Available connections after checkout: "
+          + pds.getAvailableConnectionsCount());
+      System.out.println("Borrowed connections after checkout: "
+          + pds.getBorrowedConnectionsCount());
+      // Perform a database operation
+      doSQLWork(conn);
+    }
+    
+    System.out.println("Available connections after checkin: "
+        + pds.getAvailableConnectionsCount());
+    System.out.println("Borrowed connections after checkin: "
+        + pds.getBorrowedConnectionsCount());
+  }  
+
+  /*
+   * Selects 20 rows from the SH (Sales History) Schema that is the accessible to all 
+   * the database users of autonomous database. 
+   */
+  public static void doSQLWork(Connection conn) {
+  
+    try {
+      conn.setAutoCommit(false);
+      // Prepare a statement to execute the SQL Queries.
+      try (Statement statement = conn.createStatement() ) {
```

Comment:
These 2 try-with-resources statements can be combined into a single statement with `;` as separator.

Reply:
Done, please check the changes